module/platform/Webots/src/Webots.cpp

```diff
@@ -0,0 +1,656 @@
+                                                 "right_hip_pitch_sensor",
+                                                 "right_hip_yaw_sensor",
+                                                 "left_elbow_pitch_sensor",
+                                                 "right_elbow_pitch_sensor",
+                                                 "left_shoulder_roll_sensor",
+                                                 "left_shoulder_pitch_sensor",
+                                                 "right_shoulder_roll_sensor",
+                                                 "right_shoulder_pitch_sensor",
+                                                 "neck_yaw_sensor",
+                                                 "head_pitch_sensor",
+                                                 "accelerometer",
+                                                 "gyroscope",
+                                                 "right_camera",
+                                                 "left_camera"};
+
+        for (const auto& sensor : sensors_list) {
+            SensorTimeStep time_step_msg;
+            time_step_msg.name     = sensor;
+            time_step_msg.timeStep = time_step;
+            to_send_next.sensor_time_steps.push_back(time_step_msg);
+        }
+
+        return to_send_next;
+    }
+
+    ActuatorRequests make_acutator_request(const ServoTargets& commands, const DarwinSensors& sensors) {
+        message::platform::webots::ActuatorRequests to_send_next;
+
+        // Convert the servo targets to the ActuatorRequests
+        for (const auto& target : commands.targets) {
+            MotorPosition position_msg;
+            position_msg.name     = translate_id_servo(target.id);
+            position_msg.position = target.position;
+            to_send_next.motor_positions.push_back(position_msg);
+
+            MotorVelocity velocity_msg;
+            // We need to calculate the servo velocity to add to the velocity message
+            // (method stolen from HardwareIO.cpp)
+            // velocity = (distance / time) * CLOCK_PERIOD_FACTOR
+            const float distance =
+                utility::math::angle::difference(target.position, getDarwinServo(target.id, sensors).present_position);
+
+            NUClear::clock::duration time = target.time - NUClear::clock::now();
+            float velocity;
+            if (time.count() > 0) {
+                velocity = distance / static_cast<float>(time.count()) * CLOCK_PERIOD_FACTOR;
+            }
+            else {
+                velocity = 0;
+            }
+            velocity_msg.name     = translate_id_servo(target.id);
+            velocity_msg.velocity = velocity;
+            to_send_next.motor_velocities.push_back(velocity_msg);
+
+            MotorTorque torque_msg;
+            torque_msg.name   = translate_id_servo(target.id);
+            torque_msg.torque = target.torque;
+            to_send_next.motor_torques.push_back(torque_msg);
```

Comment:
We need to query how the robocup controller is going to use this message. If this is part of force/torque control mode then we dont need to set this message at all. If it is used to set the maximum available torque then we do need to set it (but I dont think this is the case)

https://cyberbotics.com/doc/reference/motor#force-and-torque-control